src/Domain/Services/CreateAccount.php

```diff
@@ -147,6 +147,13 @@ protected function should_create_customer_account( \WP_REST_Request $request ) {
 			return true;
 		}
 
+		// check for whether account creation is enabled at the global level.
+		$checkout = WC()->checkout();
```

Comment:
I've added our own version of this routine in f076ff062f1b63718c337111c2d705f7d76a50c0. It's in `Assets.php` temporarily, as that's where I needed it, but we should move somewhere else (service class?).

Reply:
I think for now, let's just keep it as is since we're aligning with the existing Checkout flow implementation of this option (which is also filtered).